packages/playwright/src/index.ts

```diff
@@ -632,6 +634,7 @@ class ArtifactsRecorder {
 
     this._pageSnapshotRecorder = new SnapshotRecorder(this, pageSnapshot, 'pageSnapshot', 'text/yaml', '.aria.yml', async (page, path) => {
       const ariaSnapshot = await page.locator('body').ariaSnapshot({ timeout: 5000 });
+      this._pageSnapshot ??= ariaSnapshot;
```

Comment:
This is going to be somewhat funny if you have `afterAll` as test's snapshot is going to be overridden.